packages/create-expo-module/src/create-expo-module.ts

```diff
@@ -210,14 +210,44 @@ async function getNpmTarballUrl(packageName: string, version: string = 'latest')
   return stdout.trim();
 }
 
+/**
+ * Gets expo SDK version from the local package.json.
+ */
+async function getLocalSdkVersion(): Promise<string | null> {
```

Comment:
Wouldn't `require('expo/package.json').version` be more accurate? 🤔 Currently this function doesn't return the "SDK version" because it could be a range or even a URL!